package.json

```diff
@@ -3,15 +3,14 @@
   "version": "0.1.0",
   "private": false,
   "dependencies": {
-    "@data-driven-forms/pf4-component-mapper": "2.4.8",
-    "@data-driven-forms/react-form-renderer": "2.4.8",
-    "@patternfly/react-core": "^3.158.3",
-    "@patternfly/react-table": "^2.28.47",
-    "@patternfly/react-icons": "^3.15.17",
+    "@data-driven-forms/pf4-component-mapper": "^2.5.0-beta.1",
```

Comment:
there is a stable version of DDF `2.5.3` for both the mapper and renderer. Sorry I did not notice earlier.